mpi/inner_script.sh

```diff
@@ -9,13 +9,16 @@ die () {
 BATCH_NNODES=$(flux resource list -n -o {nnodes})
 BATCH_NCORES=$(flux resource list -n -o {ncores})
 COMPILER=$1
-MPI=$2
+MPI=$(echo $2 | sed "s/\///")
 export NAME="$COMPILER"_"$MPI"
 
 test -n $COMPILER || die "COMPILER (argument 1) not set"
 test -n $MPI || die "MPI (argument 2) not set"
+if [[ $MPI == "openmpi5" ]]; then
+    module use -q /usr/global/tools/mpi/toss_4_x86_64_ib/modulefiles/Core/
+fi
 module load $COMPILER || die "Compiler $COMPILER is unavailable on $LCSCHEDCLUSTER"
-module load $MPI || die "MPI implementation $MPI is unavailable on $LCSCHEDCLUSTER"
+module load $2 || die "MPI implementation $MPI is unavailable on $LCSCHEDCLUSTER"
```

Comment:
A suggestion would be to avoid using the shell position parameters in this part of the script (just for your future sanity).

Maybe use
```sh
MPI_MODULENAME=$2
MPI=${MPI_MODULENAME///} # bash-specific
```

So `$MPI_MODULENAME` contains the MPI module name and `MPI` contains a string with `/` stripped from the name.

However, I don't think this is necessary at this point (see above)

Reply:
Well, it's _kinda_ necessary. The module name with a `/` in it can't be used in line 25 to create a directory (it'll end up trying to create, for example, a subdirectory `5` under `clang_openmpi` and fail.) 

However, I see now on line 13 I'm creating another variable called name for the compiler_mpi name. Duh! I could just `sed` that, and keep things cleaner later. Fixing.